src/overlays/actors/ovl_En_Ko/z_en_ko.c

```diff
@@ -489,23 +489,23 @@ u16 EnKo_GetTextIdAdult(PlayState* play, Actor* thisx) {
 }
 
 u16 EnKo_GetTextId(PlayState* play, Actor* thisx) {
-    u16 faceReaction;
+    u16 textId;
     EnKo* this = (EnKo*)thisx;
 
     if (ENKO_TYPE == ENKO_TYPE_CHILD_0 || ENKO_TYPE == ENKO_TYPE_CHILD_2 || ENKO_TYPE == ENKO_TYPE_CHILD_3 ||
         ENKO_TYPE == ENKO_TYPE_CHILD_4 || ENKO_TYPE == ENKO_TYPE_CHILD_7 || ENKO_TYPE == ENKO_TYPE_CHILD_8 ||
         ENKO_TYPE == ENKO_TYPE_CHILD_11) {
-        faceReaction = Text_GetFaceReaction(play, 0x13);
+        textId = Text_GetFaceReaction(play, FACE_REACTION_SET_KOKIRI_CHILD_1);
```

Comment:
Here too I think you can omit "child"
I guess it relates to the ENKO_TYPE better but I think the type enum just uses it as "generic unnamed kokiri", so for face reactions it could just be KOKIRI_1,2